src/vs/workbench/contrib/webview/browser/pre/main.js

```diff
@@ -455,11 +461,21 @@ const applyStyles = (document, body) => {
 	}
 
 	if (body) {
-		body.classList.remove('vscode-light', 'vscode-dark', 'vscode-high-contrast');
+		body.classList.remove('vscode-light', 'vscode-dark', 'vscode-high-contrast', 'reduce-motion', 'enable-motion', 'using-screen-reader');
```

Comment:
Let's use the `vscode-` prefix on these new classes to make it more clear they are from VS Code

Reply:
added 👍

Reply:
Somewhat related—now that we have `highContrast` and `highContrastLight` theme types, do we need to split the `vscode-high-contrast` class into two for any reason? I'm not yet familiar with this code so just trying to make sure we're not breaking anything for webviews. cc @hawkticehurst

Reply:
@daviddossett To avoid regressions, I think the safest option would be to add a new `vscode-high-contrast-light` class that is set in addition to `vscode-high-contrast` (that way any existing webviews that have not written high contrast light specific css will fall back to their existing high contrast styling)

With my proposal, the classes in the webview would be:

- high contrast dark: `vscode-high-contrast`
- high contrast light: `vscode-high-contrast vscode-high-contrast-light`

Does that make sense?

Reply:
I think this makes sense. It mirrors the approach we've taken with the theme types to avoid breaking the existing api. We've done the same with classes in core by keeping `.hc-black`and adding `.hc-light`. 

@hawkticehurst FYI
@JacksonKearl is this something you could add next iteration?